Engineering guidebook improvements

ArtisticRoomba opened the pull request:
## About the PR
Improved the engineering guidebook following review in #33062.

Adds a section for upgrading solar panels in the solar panel entry.

## Why / Balance
Things need to have more clarity, new info (hey I said I would be keeping up with this godo).

## Technical details
Adjusted YAML formatting

## Media
![image](https://github.com/user-attachments/assets/ddb7f83d-99af-4881-b234-7956b111918f)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

**Changelog**
:cl:
- add: The Solar Panels section of the Engineering Guidebook has been updated to include information on upgrading solar panels, as well as how much each variant produces.
- tweak: The Tesla Engine section of the Engineering Guidebook has been slightly reorganized to better outline the differences between Grounding Rods and Tesla Coils, as well as tips on how to use them.
- add: The Access Configurator section of the Engineering Guidebook has been updated to reflect the recent addition of the Authentication Disruptor, as well as how to fix access-broken doors and equipment the easy way.

ScarKy0 commented on Resources/ServerInfo/Guidebook/Engineering/Spacing.xml:
```diff
@@ -6,14 +6,14 @@
   2. Repressurize the area.
 
-  - [textlink="Air vents" link="AirVent"] enter pressure lockout when a room is spaced, so you'll need to override the vents to repressurize the area. You can do this by setting the connected [textlink="air alarm" link="AirAlarms"] to fill, or by using a screwdriver on a vent to manually override it temporarily.
+  - [textlink="Air vents" link="AirVent"] enter pressure lockout in order not to lose more air to space, so you'll need to override the vents to repressurize the area. You can do this by setting the connected [textlink="air alarm" link="AirAlarms"] to fill, or by using a screwdriver on a vent to manually override it temporarily.
```
```suggestion
  - [textlink="Air vents" link="AirVent"] enter pressure lockout in order not to lose more air to spacing, so you'll need to override the vents to repressurize the area. You can do this by setting the connected [textlink="air alarm" link="AirAlarms"] to fill, or by using a screwdriver on a vent to manually override it temporarily.
```
Unsure but looks better

ScarKy0 commented on Resources/ServerInfo/Guidebook/Engineering/SingularityEngine.xml:
```diff
@@ -62,11 +62,11 @@
   The maximum power the radiation collector can produce is determined by:
-  - The amount of radiation it is capturing (which is effectively the Singularity's power level),
+  - The amount of radiation it is capturing (which is effectively the Singularity's power level)
   - and the amount of plasma it has in its connected tank.
```
```suggestion
  - The amount of plasma it has in its connected tank
```

Unsure if this is correct but it looks better in my eyes

ScarKy0 commented on Resources/ServerInfo/Guidebook/Engineering/Spacing.xml:
```diff
@@ -6,14 +6,14 @@
   Fixing spacing generally follows two simple steps:
 
-  1. Identify the area that has been spaced, and [textlink="seal the hole." link="ExpandingRepairingStation"]
+  1. Identify the area that has been space and [textlink="seal the hole." link="ExpandingRepairingStation"]
```
```suggestion
  1. Identify the area that has been spaced and [textlink="seal the hole." link="ExpandingRepairingStation"]
```
You ate something

ArtisticRoomba replied:
I was tired

ScarKy0 reviewed (changes requested):
Get reviewed, nerd
Just some minor nitpicks

ScarKy0 commented:
I pushed the fixes that I found since you asked on discord, if you want to revert the optional ones feel free to.

ScarKy0 reviewed (approved):
Lookings all fine to me now.

ArtisticRoomba commented:
Thanks for the catch, I'll write those up later. 

I often write twice during incoherent rambling (the entire guidebook) so thanks for catching it.

ArtisticRoomba commented:
I've updated parts of the Tesla Engine page to be more clear, added some solar panel info that I felt was needed, as well as outlined the recent changes to the Auth Disruptor.

beck-thompson reviewed (approved):
Tested in game everything seems to work!
